test/e2e-test-application/src/luigi-config/extended/auth.js

```diff
@@ -35,12 +35,21 @@ class Auth {
     // and run docker-compose up. Default user: Luigi , password: pwd
     authority: 'http://localhost:4011',
     logoutUrl: 'http://localhost:4011/connect/endsession',
-    client_id: 'implicit-mock-client',
     scope: 'openid profile email',
     // optional parameters
     // redirect_uri: '',
     // post_logout_redirect_uri: '/logout.html',
+    // options settings
     // automaticSilentRenew: true,
+
+    // for PKCE flow
+    client_id: 'authorisation-code-pkce-mock-client', // oidc-mockserver client id
+    response_type: "code", for PKCE
```

Comment:
```suggestion
    response_type: "code", // for PKCE
```
Sorry, missed that one.